publication-commons/src/test/java/no/unit/nva/publication/service/impl/MigrationTests.java

```diff
@@ -224,6 +233,29 @@ void shouldNotDoOnlineLookupWhenMigrating() {
         verify(uriRetriever, never()).getRawContent(ArgumentMatchers.any(), ArgumentMatchers.any());
     }
 
+    @Test
```

Comment:
Is this to be removed once migration is done? We should mark such things as deprecated